### PR TITLE
Meta function for aten.sort and aten.scatter*

### PR DESCRIPTION
**Pull request by SherlockNoMad**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #88595
* __->__ #88705

### CODE REVIEW

**Comment by ezyang** [comment]
oh no this is my bad, I forgot to claim scatter: https://github.com/pytorch/pytorch/pull/88595

**Comment by ezyang** [comment]
there's a decent amount of non-overlap, so I guess we should merge this one too

**Review by ezyang — APPROVED** [review]

**Comment by ezyang** [comment]
gonna setup a land train for this

File: torch/_meta_registrations.py [hunk 1]
```diff
@@ -1670,6 +1670,71 @@ def meta_scatter_add(self, dim, index, src):
     return self.new_empty(self.shape)
 
 
+@register_meta(aten.scatter_add_)
+def meta_scatter_add_(self, dim, index, src):
+    scatter_meta_impl(self, dim, index, src, "add")
+    return self
+
+
+@register_meta(aten.scatter)
```
Comment (anjali411) [inline torch/_meta_registrations.py:1679]: shouldn't we register this for a specific overload?

**Pull request closed without merging**